### ISSUE TITLE
text-align CSS property not working inside flex parent.

### ISSUE THREAD

**Issue by virtuallyunknown** [vercel/satori#510]
# Bug report

## Description / Observed Behavior

I am trying to use the `tw="text-right"` tailwind class and/or the `style={{ textAlign: 'right' }}` CSS property, but they aren't working as intended. The problem isn't specific to `right`, `center` and `left` aren't working either.

## Expected Behavior

[Demo on https://play.tailwindcss.com](https://play.tailwindcss.com/ah7CumyHVj)

![Screen Shot 2023-06-12 at 09 40 31](https://github.com/vercel/satori/assets/24871108/84efb15c-6fd2-41c9-bf5e-eb757e0e321f)

## Reproduction

[Demo on https://og-playground.vercel.app.](https://og-playground.vercel.app/?share=xZHNaoQwEIBfJcxlL4Z1y5ZC2C300DforfagZtSUmEgcG0V89yaxP9BDr3sKM-T7-GBWqK1EEHCR6oORvxbQaJyZ56_nPB_mNxZHXlvNFGE_8hoNoWNVy30XNgU8FoaxX7pf-DkxUXG3KyrrZGTSw60rTYv8Ps9_LGzgpy9TcI1DaXaZ582k9R--dYgm4YQzcafajgI82h7T5nKMgj3rGLpuF1gAG2nReF3XGPakVWsEO6Tiw7b9m_z9QgZ2IGXNCGIFryR1IMJpMugwekCcHs4ZSKymFkRT6hEzwN6-q5dliIcln6bgaayh575CCYLchFsGVFbhR4daW2-dlrB9Ag)

![image](https://github.com/vercel/satori/assets/24871108/c95d3736-a94d-4669-b650-d144f60b1299)

## Additional Context

Satori version: 0.10.0
Node version: 19.9.0

---

I am not really sure what is causing this, but it seems like a pretty important feature. Submitting this issue with the hope it gets fixed, and I would really appreciate it if someone knows a workaround for this problem for the time being.

Cheers!

EDIT: I also discovered some really weird issue with [truncation](https://og-playground.vercel.app/?share=zVM9T8MwEP0r1iHUJVKDYIpKpaowMDHA2MWJz4mLY1u20zSK8t85N0LlcwQx2ffe-d3d03mEygqEAlZCHViIg8bbcdwZxoQKTvOhYAup8bjIEpZud8pjFZU1xFRWd62ZuX0XopLD1pqIJiaSDvQzybWqzUPENnwieiViQ9hVnl_OSMmrl9rbzoit1dYTdyGlXOzMNK0T_6HRL13-1CKbJsZO70khOG7OEvaAXmrbU3KjhECah_WNivjkeIWEGtt77giNeIyP52zUWrmgQhJfbzarZdL93Rrvi6yW5MT_tuSPTPnWlrcTMrAuTR6gGOG0b1Dc5HkGDaq6iVBcp0Bg2dVQSK4DZoCt3avnwaWfEftTRDqSdvu-LVFAEX2HUwaRl5TRUEO2t14LmF4B).

EDIT2: [Another example of a bug](https://og-playground.vercel.app/?share=nVKxTsMwEP0V6yTUJYgAmwUsFTMDjF2c-JK4uLZlO02jKP_OOSEyqpjqxXfvne7uPXuC2koEDi9SnQ-GsRBHja_TlGLGKlF_t972Ru6ttp6z3dCpiLtipTtUbRcJfSzLuw0clIzdFSZVcFqMhDYaL7sEzvPBvKVgm3w9m7H6d2alaY2tFQ1IK3w6USNxxg5euEzaM_pG24GoTkmJJlMRL_Ej06i1ckGFXJDVEq2TtNbj3w6btOeydJcMH_sQVTPurYlokh1J5D0amUv-MWC1gLHFBCZuPIuDD2QhtVlvKMC6qKwJwCdYVgZOGxewPhfwp5RIrPoWeCN0wALwZI_qa3TpK8RhyahPQ4reTxVK4NH3OBcQRUUVHXlnB-u1hPkH).

**Comment by Shubhdeep12** [comment]
Hey @shuding, can you please assign this issue to me? I would love to resolve this. 🚀

**Comment by shuding** [comment]
Hi @Shubhdeep12 @virtuallyunknown sorry I just noticed that this isn't a bug actually, but expected behavior in Satori. This is because Satori only supports the Flexbox layout so everything has `display: flex`. As you can see it looks exactly the same in HTML view:

![CleanShot-2023-06-19-ea7cYo53@2x](https://github.com/vercel/satori/assets/3676859/0107ec89-652d-4692-8f5d-75ebcb556e11)

**Comment by virtuallyunknown** [comment]
> Hi @Shubhdeep12 @virtuallyunknown sorry I just noticed that this isn't a bug actually, but expected behavior in Satori. This is because Satori only supports the Flexbox layout so everything has `display: flex`. As you can see it looks exactly the same in HTML view:
> 

It would be really great if you guys could improve the documentation a bit, I remember I struggled quite a bit trying to figure out stuff. For example, at first I didn't even realize that tailwind was built into satori and I thought it had to be configured on it's own. Another example is that you basically have to figure out through trial and error which tw props are supported. 

Anyway, I mean this purely as feedback, don't mean to sound rude or anything like that. I appreciate the technology you've built here. For the time being I had to move to Puppeteer (even though I didn't want to, but wasn't sure I will get a response here), but would absolutely love to try Satori again for a future project.

PS: HTML Preview doesn't work on Firefox:

![image](https://github.com/vercel/satori/assets/24871108/79b52d70-7096-43f6-90f5-a074fea71c26)

Cheers!
